content/greenlake/data-services-on-the-hpe-greenlake-platform/home.md

```diff
@@ -0,0 +1,90 @@
+2. **Cloud Data Services suites** for hybrid cloud workload lifecycle management, to enable data protection, to provide disaster recovery, to deploy self-service hybrid cloud, and to enable infrastructure analysis. These services known as [HPE GreenLake for Backup and Recovery](https://www.hpe.com/us/en/hpe-greenlake-backup-recovery.html), [HPE GreenLake for Private Cloud Business Edition](https://www.hpe.com/us/en/hpe-greenlake-private-cloud-business-edition.html), and [HPE GreenLake for Disaster Recovery](https://www.hpe.com/us/en/hpe-greenlake-disaster-recovery.html).
+
+![HPE GreenLake Data Services Catalogue](/img/data-services-catalogues.png)
+
+*The above figure displays the list of the services that are part of the Data Services on the [HPE GreenLake Cloud to Edge platform](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us&page=index.html)*
+
+## Data Services REST APIs
+
+### The history
+
+In **November 2021**, after the release of the HPE GreenLake Cloud platform, Data Services introduce the first set of APIs to manipulate resources made available under HPE GreenLake Data Ops Manager. Today, this set of API was expanded to include Block Storage, File Storage services, and current storage Family of HPE Primera, Nimble Gen5, and Alletra (6K,9K,MP). This set of APIs, was associated with **Infrastructure Platform suites**, also known as the **Data Services Cloud Console API v1.5 (March 2024)**. This existing API conforms to the **OpenAPI Standard 3.0** specifications, and the spec file can be downloaded in **either YAML or JSON OpenAPI Specification** from the following [website](https://console-us1.data.cloud.hpe.com/doc/api/v1/). The blog posts about getting started with this set of existing APIs is available in this [HPE Developer Forum](https://developer.hpe.com/blog/getting-started-with-the-hpe-data-services-cloud-console-public-rest-api/). There are also additional blog posts that contains information such as [authentication using OAuth 2.0](https://developer.hpe.com/blog/oauth2-for-hpe-greenlake-data-services-cloud-console/) to generate the required [access token](https://developer.hpe.com/blog/getting-started-with-the-hpe-data-services-cloud-console-public-rest-api/), [using openapi-generator-cli](https://developer.hpe.com/blog/get-started-building-dscc-api-client-libraries-for-python-using-openapi-generator/) to convert the DSCC API OpenAPI Specifications to [PowerShell](https://developer.hpe.com/blog/getting-started-with-the-hpe-data-services-cloud-console-powershell-sdk/) and [Python client Library](https://developer.hpe.com/blog/get-started-building-dscc-api-client-libraries-for-python-using-openapi-generator/), a [PowerShell toolkit](https://developer.hpe.com/blog/new-powershell-toolkit-available-for-managing-hpe-data-services-cloud-console/) and [an example of ansible playbooks](https://developer.hpe.com/blog/automating-operations-on-dscc-using-ansible-playbooks/) to manage this storage sources.
```

Comment:
after the release of the HPE GreenLake edge-to-cloud platform, Data Services introduce....